packages/components/modal/src/modal.tsx

```diff
@@ -1,11 +1,15 @@
 import { Heading } from "@buoysoftware/anchor-typography";
-import { Box } from "@buoysoftware/anchor-layout";
+import { Box, Flex } from "@buoysoftware/anchor-layout";
 import ReactModal, { Props as ReactModalProps } from "react-modal";
 import { rgba } from "polished";
 import { useTheme } from "@buoysoftware/anchor-theme";
 import { get } from "styled-system";
 
-ReactModal.setAppElement("#root");
+import { CloseModal } from "./close_modal";
+
+if (!(process.env.NODE_ENV === "test")) {
+  ReactModal.setAppElement("#root");
+}
```

Comment:
While this is certainly not ideal in terms of cleanliness, it does keep users from having to set up specific jest configs to be able to test components that leverage `@buoysoftware/anchor-ui/modal`.

I ran this by @acostanzo, and a potential solution we started to spitball around was the possibility of making an exportable jest config/mock that wraps the Modal component and passes the `ariaHideApp={false}` prop as well as sets up the `spyOn` for `setAppElement`. 

WDUT?

Reply:
I dig this!